.github/workflows/frontend-build-check.yaml

```
@@ -44,7 +44,7 @@ jobs:
         run: yarn test
       - name: Localization build
```

Suggestion: fail CI if extracted strings don't match?
```suggestion
      - name: Check extracted strings
        working-directory: frontend
        run: yarn localize:extract && git diff-index HEAD --
      - name: Localization build
```

---

Nice idea!